cpp/oneapi/dal/table/backend/convert/copy_convert_dpc.cpp

```diff
@@ -105,33 +105,41 @@ sycl::event copy_convert(sycl::queue& queue,
                          const std::int64_t* out_strides,
                          const shape_t& shape,
                          const std::vector<sycl::event>& deps) {
-    sycl::event::wait_and_throw(deps);
-    const detail::data_parallel_policy policy{ queue };
     auto hpolicy = detail::host_policy::get_default();
+    const detail::data_parallel_policy dpolicy{ queue };
+
```

Comment:
Also, it might be better to add some asserts/exceptions to check that input pointers are not NULL.

Reply:
Discussed